packages/store/src/configs/messages.config.ts

```diff
@@ -31,5 +33,8 @@ export const CONFIG_MESSAGES: ObjectKeyMap<Function> = {
   [VALIDATION_CODE.SELECT_NOT_CONNECTED]: () =>
     `You can't use @Select your instance is frozen`,
   [VALIDATION_CODE.SELECT_CLASS_NOT_EXTENSIBLE]: () =>
-    `Your class is not extensible for @Select`
+    `Your class is not extensible for @Select`,
```

Comment:
`@Select() decorator cannot be used with non-extensible classes`

Reply:
done